tests/tests/pushdown.rs

```diff
@@ -146,3 +146,53 @@ fn pushdown(mut conn: PgConnection) {
         );
     }
 }
+
+#[rstest]
+fn pushdown_is_not_null(mut conn: PgConnection) {
```

Comment:
We'll want some more complete tests that also ensure the results from the query are correct.

Reply:
sure.